### DISCUSSION TITLE
Label size

### DISCUSSION THREAD

**Discussion by ArcusVonSinus** [cgiesche/streamdeck-homeassistant#252] [Feature Requests]
It would be neat if you could somehow change the label size. Especially for things like temperature. Now it uses only a small fraction of the button. 
(The change of general appearance would be awesome - change color, size, font, ...)

**Comment by ArcusVonSinus** [comment]
I have missed the workaround mentioned here: https://github.com/cgiesche/streamdeck-homeassistant/discussions/219. It would still be nice to have a proper label support.

**Comment by cgiesche** [comment]
The workaround should hopefully do for now.
I am currently working on a complete rewrite of the "image generation" component. This will allow easier feature implementation in the future. I will keep your wish in mind :)

**Reply by ArcusVonSinus** [reply]
Love to hear that there is a development in progress :-). Is there also a plan for the StreamDeck+ knob image?
